src/darwin/Framework/CHIP/templates/CHIPClustersObjc-src.zapt

```diff
@@ -63,6 +52,32 @@ using namespace chip::app::Clusters;
       {{/last}}
     {{/chip_cluster_command_arguments}}
 
+    {{#unless mustUseTimedInvoke}}
+    if (self.isGroupCluster) {
+      auto error = DispatchGroupCommand(request, self.groupId, self.fabricIndex);
+      dispatch_async(self.callbackQueue, ^{
+        {{#if hasSpecificResponse}}
+        completionHandler(nil, error);
```

Comment:
This is pretty weird: it will call `completionHandler` with `(nil, nil)` on success.

It makes no sense, normally, to send group commands for things that expect a response per spec....  But I guess nothing prohibits it?  We need to really carefully document this behavior.

Reply:
Would you prefer if for the moment the code fails for commands that expects a response instead ?